django/core/files/move.py

```diff
@@ -82,12 +81,12 @@ def file_move_safe(
 
     try:
         copystat(old_file_name, new_file_name)
-    except PermissionError as e:
+    except PermissionError:
         # Certain filesystems (e.g. CIFS) fail to copy the file's metadata if
         # the type of the destination filesystem isn't the same as the source
-        # filesystem; ignore that.
-        if e.errno != errno.EPERM:
-            raise
+        # filesystem. This also happens with some SELinux-enabled systems.
+        # Ignore that, but always set basic permissions.
+        copymode(old_file_name, new_file_name)
```

Comment:
In practice this may again fail with a `PermissionError`. This might happen if the mount maps users in suboptimal ways. So after copying the file you'd no longer have the rights to change it at all. To keep backwards compatibility we'd still might want to ignore the `EPERM` here.

Reply:
I agree. I will add `try` here